R/geom-curve.r

```diff
@@ -0,0 +1,66 @@
+#' Single curved line segments.
+#'
+#' @section Aesthetics:
+#' \Sexpr[results=rd,stage=build]{ggplot2:::rd_aesthetics("geom", "curve")}
+#'
+#' @inheritParams grid::curveGrob
```

Comment:
You need to keep the previous `@inheritParams` and this `@inheritParams grid::curveGrob`. Then you don't need to document the individual arguments below because they'll be automatically pulled from curveGrob's documentation